doc/api/events.md

```diff
@@ -1337,7 +1337,7 @@ changes:
 -->
 
 * `emitter` {EventEmitter}
-* `name` {string}
+* `name` {string|symbol}
```

Comment:
I believe Symbol is capitalized, but I might be wrong.

Also, IIRC there should be spaces between the pipe

Reply:
`symbol` is lowercase here to refer to the primitive symbol type.

`Symbol` is incorrect; this would be an object with a prototype of `Symbol.prototype` (similar to `string` versus `String` _etc._)

Reply:
The convention here appears to be `{A|B}` in markdown documentation, but `{A | B}` in JSDoc annotations.

Reply:
Great, thanks